docs(compute): update persistent_tpm_and_uefi_state.md (#912)
kubevirt/user-guide: docs/compute/persistent_tpm_and_uefi_state.md

            persistent: true
```

> In order for the persistent tpm volume to be created successfully you must ensure your storage classes and storage profiles are configured correctly.  

Reviewer: ```suggestion
In order for the persistent tpm volume to be created successfully you must ensure your storage classes and [storage profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) are configured correctly.  
```

The persistent tpm volume will be created with the below access mode if one of the constraints for the access mode is true.

> RWX:

Reviewer: ```suggestion
RWX (ReadWriteMany):
```

> - the respective storage profile has any claim property set with RWX access mode

Reviewer: ```suggestion
- the respective storage profile has any claim property set with Filesystem volume mode and RWX access mode
```

- the kubevirt cluster config has `VMStateStorageClass` set and the storage profile does not exist or the storage profile exists but has no claim property sets

> RWO:

Reviewer: ```suggestion
RWO (ReadWriteOnce):
```

> - the respective storage profile has only claim property sets with RWO access mode

Reviewer: ```suggestion
- the respective storage profile has claim property sets with Filesystem volume mode and RWO access mode but not RWX
```

> - the kubevirt cluster config has `VMStateStorageClass` unset and the storage profile does not exist or the storage profile exists but has no claim property sets

Reviewer: ```suggestion
- the kubevirt cluster config has `VMStateStorageClass` **unset** and the storage profile does not exist or the storage profile exists but has no claim property sets
```

>     * the respective storage profile has claim property sets with Filesystem volume mode and RWO access mode but not RWX.

Reviewer 2: nit - set

Author: > the respective storage profile has claim property sets with Filesystem volume mode and RWO access mode but not RWX.

I believe if we change *sets* to *set* it will be incorrect because a [StorageProfile](https://pkg.go.dev/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1#StorageProfileSpec) has claimPropertySets where each claimPropertySet in the claimPropertySets has volume mode Filesystem and RWO mode.

Author: I could change this to the below, thoughts?
> the respective storage profile has `claimPropertySets` where all `claimPropertySet` in `claimPropertySets` have Filesystem volume mode and RWO access mode but not RWX.

>     * the kubevirt cluster config has `VMStateStorageClass` **unset** and the storage profile does not exist or the storage profile exists but has no claim property sets.

Reviewer 2: nit - set

Reviewer: I agree it's debatable above, but less so here. An empty claimPropertySets section has no claim property set... Maybe reword to avoid the confusion, like "has no claim property set defined" or "has an empty claimPropertySets list"?
